modules/exploits/linux/http/vmware_view_planner_4_6_uploadlog_rce.rb

```diff
@@ -0,0 +1,144 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Remote
+
+  # "Shotgun" approach to writing JSP
```

Comment:
This looks familiar and out of place.

Reply:
Woops, yeah taken from another module, I should have removed this, let me adjust this.

Reply:
Also noted the Ranking is wrong, will update that as well.

Reply:
Fixed this error and updated the exploit ranking with https://github.com/rapid7/metasploit-framework/pull/14875/commits/d3918f5054b14b263b89dc6511f64c8fbf7d59c1